projects/sport-data-valley/manual_seeds/training_responses.rb

```diff
@@ -0,0 +1,248 @@
+# frozen_string_literal: true
```

Comment:
Again I would prefer this to be in a class or a module.

You can do something basic, like

```ruby
srand(123)
require_relative 'helpers/people_helper' # Make sure PeopleHelper is defined in our scope

class TrainingResponsesSeeds
  include PeopleHelper
  class << self
     def run!
        # do whatever
     end

    private
    
    # all your private methods here

  end
end

# Then as final line in the file you have this, so that it runs when you load the file (just make sure it's not in an autoloaded path (which I think is fine currently):
TrainingResponsesSeeds.run!
```

And then we keep a very clean namespace even when running seeds, but we can still load seeds on demand that are not autoloaded everytime we start the application.